apps/files/lib/Command/TransferOwnership.php

```
@@ -274,7 +283,18 @@ private function restoreShares(OutputInterface $output) {
 					if ($shareMountPoint) {
 						$this->mountManager->removeMount($shareMountPoint->getMountPoint());
 					}
-					$this->shareManager->deleteShare($share);
+					try {
+						$this->shareManager->deleteShare($share);
+					} catch (\OCP\Files\NotFoundException $e) {
+						/**
+						 * This is useful when transfer ownership is made on files
+						 * shared to other users and the files are encrypted with
+						 * master key. The method getPath from \OC\Files\Filesystem tries
+						 * to call getPath using defaultInstance variable which
+						 * is null. Hence this causes our integration tests to break.
+						 */
```

Thanks for the explanation. I think we shouldn't catch and ignore an exception like this just because of integration tests.

I was fine with your other workaround in the other method when `instance` is null but here I think it's a bit too much.
Do you know which test fails ? Maybe there's a setupFS missing somewhere to initialize that default instance ?

From what I see the `defaultInstance` is created when calling `Filesystem::init()`.
Can you debug and compare why that one is properly set when running OCC manually and not when running integration tests ? There shouldn't be any difference...